packages/kit/src/views/Staking/components/ProtocolDetails/GridItemV2.tsx

```
@@ -180,7 +251,82 @@ export function GridItem({
+                  variant="tertiary"
+                />
+              }
+              renderContent={
+                <RewardAmountPopoverContent
+                  tooltip={tooltip}
+                  onHistory={onHistory}
+                />
+              }
+            />
+          );
+        case 'text':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)